pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -790,4 +793,13 @@ public static void deregister() throws SQLException {
   public static boolean isRegistered() {
     return registeredDriver != null;
   }
+
+  /**
+   * StreamWrapper needs this and instead of passing this all the way down, it makes more sense
+   * to simply get the one instance this way
+   * @return instance of LazyCleaner
+   */
+  public static LazyCleaner getCleanerInstance() {
+    return CLEANER;
+  }
```

Comment:
Would it be better to move the "default cleaner" to `LazyCleaner` itself?
Then it would be easier to clarify that `LazyCleaner` is "driver-internal" class.

Reply:
Certainly encapsulates it better. I'll make that change.

Reply:
moved